examples/dht_server.rs

```diff
@@ -228,30 +228,35 @@ fn add_server_main_loop(base_selector: IoFuture<()>, server_obj: &Server) -> IoF
             println!("main_loop_wakeup");
             // flag for fast bootstrapping
             if bootstrap_fast {
+                server_obj_c.dht_main_loop()
+            } else {
+                bootstrap_fast = true;
                 // args to main loop, all value is seconds
-                let args = DhtMainLoopArgs {
+                let args = ConfigArgs {
                     kill_node_timeout: 182,
                     ping_timeout: 5,
-                    ping_interval: 60,
+                    ping_interval: 0,
                     bad_node_timeout: 162,
-                    nodes_req_interval: 20,
-                    nat_ping_req_interval: 3,
+                    nodes_req_interval: 0,
+                    nat_ping_req_interval: 0,
                 };
 
-                server_obj_c.dht_main_loop(args)
-            } else {
-                bootstrap_fast = true;
+                server_obj_c.set_config_values(args);
```

Comment:
No need to set config values each wakeup. You may do that before you start the server

Reply:
Btw, do we need to have these as variables? Perhaps we can just use constants? This will require to change `Instant` to `SystemTyme` for the tests, but there are some benefits of this change anyway. So is there some scenario where setting these custom (not default) variables might be usefull?

Reply:
Yes. bootstrap_fast is a flag for first calling of dht_main_loop().
The separation of first call and other calls is for fast bootstrapping.
So set config is called just two times( first call, and second call)

Reply:
I also think about that. These constants may not changed per copy of toxcore.

Reply:
resolved